src/Forms/Settings.cs

```diff
@@ -2553,5 +2558,18 @@ private void linkLabelOpenDictionaryFolder_LinkClicked(object sender, LinkLabelL
             System.Diagnostics.Process.Start(dictionaryFolder);
         }
 
+        private void textBoxVlcPath_MouseLeave(object sender, EventArgs e)
+        {
+            try
+            {
+                var path = textBoxVlcPath.Text.Trim('\"');
+                if (path.Length > 3 && Path.IsPathRooted(path) && Path.GetFileName(path).Equals("vlc.exe", StringComparison.OrdinalIgnoreCase) && File.Exists(path))
+                {

```

Comment:
I guess the portable stuff is only relevant for windows users, right?
Also, I've not succeeded in making VLC work as video player on linux (or osx - don't have a mac).